### DISCUSSION TITLE
How can I stop the autogenerated examples?

### DISCUSSION THREAD

**Discussion by tjperry07** [PaloAltoNetworks/docusaurus-openapi-docs#1034] [Q&A]
I want to stop these from appearing in the result. The only way I can see so far is to edit the plugin directly. I used to be able to delete them, but the new refactor removed them from the MDX. 

Example (auto)

<img width="643" alt="Screenshot 2024-12-03 at 3 19 07 PM" src="https://github.com/user-attachments/assets/befe0535-100f-48b6-9caa-364ed75dbcbb">

**Comment by sserrata** [comment]
Hi @tjperry07, currently there's no plugin option to suppress them but you should be able to swizzle the `ResponseExamples` theme component to comment them out.
